docs/book/src/user/quick-start.md

```diff
@@ -357,50 +357,28 @@ See the [AWS provider prerequisites] document for more details.
 {{#/tab }}
 {{#tab Azure}}
 
+NOTE: `CLUSTER_NAME` can only include letters, numbers, and hyphens and can't be longer than 44 characters to allow for cluster name 
+to be used as a prefix for VMs and other resources that have [Azure naming rules and restrictions](https://docs.microsoft.com/en-us/azure/azure-resource-manager/management/resource-name-rules)
+
 ```bash
 export CLUSTER_NAME="capi-quickstart"
-# Name of the virtual network in which to provision the cluster.
```

Comment:
these were used in the azure json var below and aren't required anymore (the templates have defaults)